apps/tangle-website/src/styles/globals.css

```diff
@@ -3,16 +3,16 @@
 }
 
 /* Headings, sub-headings, card texts */
-.heading-1 {
-  @apply text-[34px] leading-[45.9px] tracking-[-0.011em] /** small screen */
-  md:text-[84px] md:leading-[113px] /** medium screen */
+.heading {
+  @apply text-[44px] leading-[52.8px] tracking-[-0.011em] /** small screen */
+  md:text-[64px] md:leading-[80px] /** medium screen */
   font-black;
 }
 
-.sub-heading-1 {
-  @apply text-[16px] leading-[21.6px] /** small screen */
-  md:text-[18px] md:leading-[27px] /** medium screen */
-  font-medium;
+.section-description {
+  @apply text-[16px] leading-[25.6px] /** small screen */
+  md:text-[24px] md:leading-[40px] /** medium screen */
+  font-medium text-mono-140;
 }
```

Comment:
Just ignore the font styles right now! We have a separate PR for adding the font styles in the webb-ui-kit.